setup.sh

```diff
@@ -260,6 +260,12 @@ setup_osie() (
 			curl -fsSL 'https://tinkerbell-oss.s3.amazonaws.com/osie-uploads/latest.tar.gz' -o ./osie.tar.gz
 			tar -zxf osie.tar.gz
 		else
+			if [ ! -f "$TB_OSIE_TAR" ]; then
+				echo "$ERR osie tar not found in the given location $TB_OSIE_TAR"
+				exit 1
+			else
+				echo "$INFO extracting osie tar"
```

Comment:
Please remove the "else" and move this echo statement to appear just before the `tar`. This will make the code easier to follow.